### ISSUE TITLE
Pure Function to render html

### ISSUE THREAD

**Issue by chungchi300** [unlayer/react-email-editor#111]
We store the design into the database in JSON with the Admin portal. How to use a pure function to render related html when I use it in the front end?

```
import {designToHtml} from 'react-email-editor';

function demo({design}){
          <div

              dangerouslySetInnerHTML={{
                __html: designToHtml(design)
              }}
            />

}

```

**Comment by lidrissi** [comment]
i dont think there is a solution, for us we store both html and json design into the database

**Comment by chungchi300** [comment]
Agree, I also store both html and json as a workaround. And I think it increases the maintainability when we follow Single source of truth if we can store the JSON only and use the render function

**Comment by chungchi300** [comment]
even an async render function is cool too.

**Comment by adeelraza** [comment]
@chungchi300 we do have a REST API that you can use to convert JSON to HTML. Check: https://docs.unlayer.com/reference#render
